src/backend/distributed/commands/call.c

```diff
@@ -99,9 +100,7 @@ CallFuncExprRemotely(CallStmt *callStmt, DistObjectCacheEntry *procedure,
 		return false;
 	}
 
-	CitusTableCacheEntry *distTable = GetCitusTableCacheEntry(colocatedRelationId);
-	Var *partitionColumn = distTable->partitionColumn;
-	if (partitionColumn == NULL)
+	if (IsReferenceTable(colocatedRelationId))
```

Comment:
Is this the only place where use `if (partitionColumn == NULL)` to detect reference tables? It'd be great if we can update all